backend/src/bin/jobs/index.ts

```diff
@@ -16,9 +14,7 @@ const jobs: CrowdJob[] = [
   discordCoordinator,
   slackCoordinator,
   weeklyAnalyticsEmailsCoordinator,
-  checkMemberScoresCoordinator,
-  checkMembersToMergeCoordinator,
-  checkSqsQueues,
```

Comment:
`checkSqsQueues` was removed - was this intentional?

Reply:
Nope. But I wonder how it works anyway when doing `clean-start-dev` 🤔

Reply:
Locally `checkSqsQueues` shouldn't work since slack token to post slack messages is not configured. I was asking if you intentionally removed checkSqsQueues job here otherwise please put it back in :D I think it was a merge issue.

Reply:
Ah, ok! Yeah, putting it back